### ISSUE TITLE
panic in 1.1.0

### ISSUE THREAD

**Issue by ry** [denoland/deno#6272]
```
> RUST_BACKTRACE=1 deno run --allow-net --unstable https://gist.githubusercontent.com/alexanderson1993/d75444f026ed8344aefa6700ba426430/raw/293fd38fb4e9bf4fff5d81a7a918c9753d0b5f34/index.tsx
Visit http://localhost:8000
thread 'main' panicked at 'Can't downcast ErrBox(JSError(JSError { message: "Uncaught AssertionError: Assertion failed.", source_line: None, script_resource_name: Some("$deno$/util.ts"), line_number: Some(35), start_column: Some(10), end_column: Some(11), frames: [JSStackFrame { type_name: Some("Object"), function_name: Some("assert"), method_name: Some("assert"), file_name: Some("$deno$/util.ts"), line_number: Some(35), column_number: Some(11), eval_origin: None, is_top_level: None, is_eval: false, is_native: false, is_constructor: false, is_async: false, is_promise_all: false, promise_index: Some(11) }, JSStackFrame { type_name: None, function_name: Some("buildLocalSourceFileCache"), method_name: None, file_name: Some("$deno$/compiler.ts"), line_number: Some(536), column_number: Some(5), eval_origin: None, is_top_level: None, is_eval: false, is_native: false, is_constructor: false, is_async: false, is_promise_all: false, promise_index: Some(5) }, JSStackFrame { type_name: None, function_name: Some("runtimeCompile"), method_name: None, file_name: Some("$deno$/compiler.ts"), line_number: Some(1245), column_number: Some(3), eval_origin: None, is_top_level: None, is_eval: false, is_native: false, is_constructor: false, is_async: false, is_promise_all: false, promise_index: Some(3) }, JSStackFrame { type_name: Some("global"), function_name: Some("tsCompilerOnMessage"), method_name: Some("onmessage"), file_name: Some("$deno$/compiler.ts"), line_number: Some(1364), column_number: Some(22), eval_origin: None, is_top_level: None, is_eval: false, is_native: false, is_constructor: false, is_async: false, is_promise_all: false, promise_index: Some(22) }, JSStackFrame { type_name: None, function_name: Some("workerMessageRecvCallback"), method_name: None, file_name: Some("$deno$/runtime_worker.ts"), line_number: Some(72), column_number: Some(33), eval_origin: None, is_top_level: None, is_eval: false, is_native: false, is_constructor: false, is_async: false, is_promise_all: false, promise_index: Some(33) }, JSStackFrame { type_name: Some("global"), function_name: None, method_name: None, file_name: Some("file:///Users/rld/src/deno/__anonymous__"), line_number: Some(1), column_number: Some(1), eval_origin: None, is_top_level: None, is_eval: false, is_native: false, is_constructor: false, is_async: false, is_promise_all: false, promise_index: Some(1) }], formatted_frames: ["Object.assert ($deno$/util.ts:35:11)", "buildLocalSourceFileCache ($deno$/compiler.ts:536:5)", "runtimeCompile ($deno$/compiler.ts:1245:3)", "tsCompilerOnMessage ($deno$/compiler.ts:1364:22)", "workerMessageRecvCallback ($deno$/runtime_worker.ts:72:33)", "file:///Users/rld/src/deno/__anonymous__:1:1"] })) to OpError', cli/op_error.rs:422:9
stack backtrace:
```

**Comment by wusung** [comment]
It seem to be happened in the specific environment.   It run well in my mac book laptop.

```bash
$ deno --version
deno 1.1.0
v8 8.4.300
typescript 3.9.2


$ env RUST_BACKTRACE=1 deno run --allow-net --unstable https://gist.githubusercontent.com/alexanderson1993/d75444f026ed8344aefa6700ba426430/raw/293fd38fb4e9bf4fff5d81a7a918c9753d0b5f34/index.tsx

Download https://dev.jspm.io/react-dom/server
Download https://unpkg.com/es-react
Download https://gist.githubusercontent.com/alexanderson1993/d75444f026ed8344aefa6700ba426430/raw/293fd38fb4e9bf4fff5d81a7a918c9753d0b5f34/app.tsx
Download https://unpkg.com/es-react@16.13.1
Download https://unpkg.com/es-react@16.13.1/index.js
Download https://unpkg.com/es-react@16.13.1/react.production.min-8d2700a6.js
Download https://unpkg.com/es-react@16.13.1/react.js
Download https://unpkg.com/es-react@16.13.1/checkPropTypes-0d3dc276.js
Download https://unpkg.com/es-react@16.13.1/index-ad6b2a22.js
Download https://unpkg.com/es-react@16.13.1/react-dom.js
Download https://unpkg.com/es-react@16.13.1/prop-types.js
Download https://dev.jspm.io/npm:react-dom@16.13.1/server.browser.dew.js
Download https://dev.jspm.io/npm:react-dom@16.13.1/cjs/react-dom-server.browser.development.dew.js
Download https://dev.jspm.io/npm:react@16?dew
Download https://dev.jspm.io/npm:object-assign@4?dew
Download https://dev.jspm.io/npm:prop-types@15/checkPropTypes?dew
Download https://dev.jspm.io/npm:prop-types@15.7.2/checkPropTypes.dew.js
Download https://dev.jspm.io/npm:prop-types@15.7.2/lib/ReactPropTypesSecret.dew.js
Download https://dev.jspm.io/npm:object-assign@4.1.1/index.dew.js
Download https://dev.jspm.io/npm:react@16.13.1/index.dew.js
Download https://dev.jspm.io/npm:react@16.13.1/cjs/react.development.dew.js
Compile https://gist.githubusercontent.com/alexanderson1993/d75444f026ed8344aefa6700ba426430/raw/293fd38fb4e9bf4fff5d81a7a918c9753d0b5f34/index.tsx
Visit http://localhost:8000


```

```bash
curl http://127.0.0.1:8000/
<html><head></head><body><div data-reactroot=""><h1>Hello World 🦕</h1></div>
    <script type="module" src="/bundle.js"></script>
    </body></html>⏎
```

**Comment by VanshVarshney** [comment]
i would like t handle this BUG.

**Comment by bartlomieju** [comment]
Can't repeat on latest `master`. I think it was fixed in #6300
